code/software/libs/src/start_serial/link_scripts/rosco_m68k_program.ld

```diff
@@ -38,32 +67,40 @@ SECTIONS
   .text.postinit : AT(SIZEOF(.text.init))
   {
     _postinit = .;
-    KEEP(*(.postinit))  /* KEEP() needed to "anchor" kmain reference for gc-sections */
+    KEEP(*(.postinit))  /* KEEP() "anchors" section for gc-sections */
     _postinit_end = .;
-  } > KRAM
+  } > RAM
 
   .text ALIGN(4) :
   {
     _code = .;
     *(.text*)
     *(.rodata*)
     _code_end = .;
-  } > KRAM
+  } > RAM
   
   .data ALIGN(4) :
   {
     _data_start = .;
     *(.data*)
     _data_end = .;
-  } > KRAM
+  } > RAM
+
+  /* verify initialized data ends before load address 0x28000  */
+  ASSERT((_data_end) < 0x28000,
```

Comment:
Is this correct, given that in 1.2 it is 0x40000 as per previous comment?

Reply:
Pretty sure it is.  I got this number from code, not a comment.  Also UART loader "failed" when my code got to 157KB and worked at 156KB, but I will verify 0x28000 was load address (it was the old "org" in init.S, so I think it had to be right).

Reply:
As above, KERNEL_LOAD_ADDRESS will vary.  I will mostly assume r1.2 number (but just for assert - code is PIC)

Reply:
I added _LOAD_ADDRESS and _RUN_ADDRESS (0x1000) to linker script, however I am removing that assert since now init works around it (and any limitations are due to loader stack - which is planned to be resolved).